File: springboot-mongo-atlas/src/main/java/ticket/model/Ticket.java
package ticket.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "tickets")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Ticket {

    @Id
    private String ticketId;
    private String ticketCode;
    private String name;
    private String startStation;
    private String lastStation;
    private int price;

    private int expiryTime;
    private boolean usedForEntry;
    private boolean usedForExit;

}

File: TicketBooking/src/app/Model/ticket.ts
export interface Ticket {
    ticketId: string;
    ticketCode: string;
    name: string;
    startStation: string;
    lastStation: string;
    price: number;
    expiryTime: number;
    usedForEntry: boolean;
    usedForExit:  boolean ;
  }
  